### PR TITLE
Add user session management rest API endpoint

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
This PR contains user session management rest API endpoint.
endpoints:
GET auth/v1.2/session - view all active sessions
DELETE auth/v1.2/session -delete all active sessions.
DELETE auth/v1.2/sesion/{sessionId} - delete a particular session.

Depends on:
Add user session api core component and stub #290

### CODE REVIEW

File: components/org.wso2.carbon.identity.local.auth.api.endpoint/pom.xml [hunk 1]
```diff
@@ -87,6 +87,12 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.governance</groupId>
+            <artifactId>org.wso2.carbon.identity.user.session</artifactId>
+            <scope>provided</scope>
+            <version>1.0.1</version>
```
Comment (senthalan) [inline components/org.wso2.carbon.identity.local.auth.api.endpoint/pom.xml]: Move the version as a property to the parent pom

**Comment by pamodaaw** [comment]
Depends on https://github.com/wso2-extensions/identity-governance/pull/290

File: components/org.wso2.carbon.identity.local.auth.api.endpoint/pom.xml (hunk 1 above)
Comment (Chuhaa) [reply components/org.wso2.carbon.identity.local.auth.api.endpoint/pom.xml]: Resolved with 78b6e0a

**Comment by pamodaaw** [comment]
We can close this PR since the API implementations are moved to `identity-api-user` with the PR https://github.com/wso2/identity-api-user/pull/28.

**Comment by mefarazath** [comment]
Closing this PR since the implementation has been moved with https://github.com/wso2/identity-api-user/pull/28

**Pull request closed without merging**